eng/versioning/version_client_java_files.txt

```diff
@@ -3,7 +3,6 @@
 # has been fixed this file and the update logic in update_versions.py should no longer be necessary.
 
 # The file format here should be the relative path from the root of the azure-sdk-for-java
-sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/AzureConfiguration.java
```

Comment:
This was an opportunistic fix. Apparently earlier today this file was removed from source (not renamed, deleted) and this caused the update script to fail.